runtime/service/src/test/java/org/apache/polaris/service/admin/PolarisAuthzTestBase.java

```diff
@@ -231,13 +232,9 @@ public void before(TestInfo testInfo) {
 
     RealmContext realmContext = testInfo::getDisplayName;
     QuarkusMock.installMockForType(realmContext, RealmContext.class);
+    realmContextHolder.set(realmContext);
     polarisContext = callContext.getPolarisCallContext();
 
-    ContainerRequestContext containerRequestContext = Mockito.mock(ContainerRequestContext.class);
```

Comment:
nit: isn't this for Request ID? Is this causing some downstream issues?

Reply:
@adnanhemani : I'm going to merge for now as it's a bug fix PR and I believe this particular change is fine. I'm open to following up on this on `main`.